sentry-raven.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "faraday", ">= 0.7.6"
   gem.add_dependency "certifi"
+  gem.add_dependency "binding_of_caller" if RUBY_VERSION >= '1.9.2' && !defined?(JRUBY_VERSION)
```

Comment:
The gemspec is evaluated when the gem is built, not when it's installed - `RUBY_VERSION` here will return the version of Ruby that is being used to build the gem. RubyGems doesn't support conditional dependencies.

I don't know if there's a way to maintain compatibility with Ruby 1.8.7 and also optionally depend on binding_of_caller.